src/wp-includes/version.php

```diff
@@ -30,7 +30,7 @@
  *
  * @global string $tinymce_version
  */
-$tinymce_version = '49110-20201110';
+$tinymce_version = '49110-20250701';
```

Comment:
```suggestion
$tinymce_version = '49110-20250317';
```

The most recent change was March 17.